cmd/manager/main.go

```diff
@@ -45,7 +46,11 @@ const (
 	updateNotificationSendWait = time.Duration(10) * time.Minute
 )
 
-var version = flag.Bool("version", false, "displays marketplace source commit info.")
+var (
+	version     = flag.Bool("version", false, "displays marketplace source commit info.")
+	tlsKeyPath  = flag.String("tls-key", "/etc/pki/ca-trust/extracted/pem/ca-bundle.crt", "Path to use for private key (requires tls-cert)")
+	tlsCertPath = flag.String("tls-cert", "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem", "Path to use for certificate key (requires tls-key)")
```

Comment:
Please use the constants defined in the `certificateauthority` package for the defaults.